data/events/scripts/monster.lua

```diff
@@ -0,0 +1,37 @@
+function Monster:onDropLoot(corpse)
+	if configManager.getNumber(configKeys.RATE_LOOT) == 0 then
+		corpse:decay()
```

Comment:
As far as I can tell the corpse is already decaying: https://github.com/otland/forgottenserver/blob/32688aceee19d175a72f61a65743613bb45288e5/src/creature.cpp#L722
So this would not be needed.

Reply:
yeah, when I was creating it I tested it and corspes were already decaying, so both decays were useless, anyways I did it just because old code was like that.
I'm going to remove it.